opendrift/models/basemodel/init.py

```diff
@@ -300,6 +300,76 @@ def __init__(self,
         self.elements_scheduled_time = np.array([])
         self.env = Environment(self.required_variables)
 
+    def add_reader(self, readers, variables=None, first=False):
```

Comment:
I copied add_reader into basemodel.Init, but perhaps it should be stored under Init.Environment? Anyway, we'd probably like to keep old syntax: o.add_reader()

Reply:
Yes, I thought it was already present in Init.Environment?

Reply:
Ok, good, I will check tomorrow